src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFOutline.java

```diff
@@ -173,12 +177,34 @@ public void copyOutline(
                 // This will remove the old destination info.
                 // Like navigating to the old/original document page number.
             }
-            item.getCOSObject().setItem(COSName.FIRST, item.getFirstChild());
-            item.getCOSObject().setItem(COSName.LAST, item.getLastChild());
+            item.getCOSObject().setItem(COSName.FIRST, removeNullObject(item.getFirstChild()));
+            item.getCOSObject().setItem(COSName.LAST, removeNullObject(item.getLastChild()));
             destLastOutlineItem.addLast(item);
         }
     }
 
+    public PDOutlineItem removeNullObject(PDOutlineItem outline) {
+        if (outline == null || outline.getCOSObject() == null) {
```

Comment:
For all the null checks in this class, Can we use method from Objects class https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html.  ?